moncli/entities/item.py

```diff
@@ -726,6 +732,31 @@ def change_column_value(self, column_value = None, get_column_values: bool = Non
                 updates : `moncli.entities.update.Update`
                     The item's updates.
         """
+        if id and title : 
+            raise ItemError(
+                'change_column_value_too_many_parameters',
+                self.id,
+                'Cannot use both "id" and "title" parameters.'
+            )
+        if not (id or title):
+            if isinstance(column_value, ColumnValue):
+                pass
+            else:
+                raise ItemError(
+                    'invalid_column_value',
+                    self.id,
+                    'Column value must be a properly formatted dict or str when using "id" or "title" parameters.'
+                )
+        if id or title:
+            if isinstance(column_value,(dict,str)):
+                if title:
+                    column_id= self.column_values[title].id
+            else: 
+                raise ItemError(
+                    'invalid_column_value_entity',
+                    self.id,
+                    'Column Value must be a valid entities.column_value.ColumnValue instance when not using "id" or "title" parameters.'
+                )
```

Comment:
I can't seem to get to it from here, but there looks to be something missing below.  

Lines 768-771 are no longer needed, as they were old checks.  
Line 772 should instead be **if isinstance(column_value, en.cv.ColumnValue):**